spring-boot-project/spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -4311,48 +4329,38 @@ pattern). If you do not want this behavior, add the following line to your
 ----
 
 
-
 [[boot-features-spring-data-neo4j-repositories]]
 ==== Spring Data Neo4j Repositories
-Spring Data includes repository support for Neo4j.
+Spring Data includes https://projects.spring.io/spring-data-neo4j/[repository support] 
+for Neo4j.
 
-In fact, both Spring Data JPA and Spring Data Neo4j share the same common infrastructure.
-You could take the JPA example from earlier and, assuming that `City` is now a Neo4j OGM
-`@NodeEntity` rather than a JPA `@Entity`, it works in the same way.
-
-TIP: You can customize entity scanning locations by using the `@EntityScan` annotation.
-
-To enable repository support (and optionally support for `@Transactional`), add the
-following two annotations to your Spring configuration:
-
-[source,java,indent=0]
-----
-	@EnableNeo4jRepositories(basePackages = "com.example.myapp.repository")
-	@EnableTransactionManagement
-----
-
-==== Repository Example
-
-The following example shows an interface definition for a Neo4j repository:
+Spring Data Neo4j shares the common infrastructure with Spring Data JPA as many other
+Spring Data modules do. You could take the JPA example from earlier and define
+`City` as Neo4j OGM `@NodeEntity` rather than JPA `@Entity` and the repository
+abstraction works in the same way, as shown in the following example:
 
 [source,java,indent=0]
 ----
 	package com.example.myapp.domain;
 
-	import org.springframework.data.domain.*;
-	import org.springframework.data.repository.*;
+	import java.util.Optional;
 
-	public interface CityRepository extends GraphRepository<City> {
+	import org.springframework.data.neo4j.repository.*;
 
-		Page<City> findAll(Pageable pageable);
+	public interface CityRepository extends Neo4jRepository<City, Long> {
 
-		City findByNameAndCountry(String name, String country);
+		Optional<City> findOneByNameAndState(String name, String state);
 
 	}
 ----
 
-TIP: For complete details of Spring Data Neo4j, including its rich object mapping
-technologies, refer to the https://projects.spring.io/spring-data-neo4j/[reference
+The `spring-boot-starter-data-neo4j` "`Starter`" enables the repository support as well 
+as transaction management. You can customize the locations to look for repositories and
+entities by using `@EnableNeo4jRepositories` and `@EntityScan` respectively on a 
+`@Configuration`-bean.
+
+TIP: For complete details of Spring Data Neo4j, including its object mapping
+technologies, refer to the https://docs.spring.io/spring-data/neo4j/docs/current/reference/html/[reference
```

Comment:
We're not pointing to `current` on purpose as it won't be accurate. This link is incorrect in the Spring Boot 2.0.x documentation for instance as we're not using the "current" version. This is the main reason we use the projects page link (see others Spring Data section for consistency).

I am not against reconsidering this but I'd prefer to do so consistently and in such a way that the version is correct.